gitlab/datadog_checks/gitlab/gitlab.py

```diff
@@ -95,6 +98,38 @@ def _service_check_tags(self, url):
         gitlab_port = 443 if parsed_url.scheme == 'https' else (parsed_url.port or 80)
         return ['gitlab_host:{}'.format(gitlab_host), 'gitlab_port:{}'.format(gitlab_port)]
 
+    def _get_api_token(self, url):
```

Comment:
This isn't used since users have the `api_token` config, but if anyone thinks it would be a good idea to try to create one if the user does not have this value, then I can add it back in as a second way to get the token.